.buildkite/setup_al2.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -eu
+
+source ./.buildkite/al2env.sh
+
+echo "Creating necessary directories"
```

Comment:
I'd recommend removing all the `echo` statements and instead using `set -x` at the top.  This option will cause every command to be echoed out before it is run.

Reply:
Done